lib/libc/stdio/printfcommon.h

```diff
@@ -279,82 +279,131 @@ __ujtoa(uintmax_t val, CHAR *endp, int base, int octzero, const char *xdigs)
 	return (cp);
 }
 
-#ifdef __CHERI_PURE_CAPABILITY__
-/*
- * Print the pointer details.  Format matches the kernel register dump format:
- * v:0 s:0 p:00000000 b:0000000000000000 l:0000000000000000 o:0 t:0
- *  v	tag (valid)
- *  s	sealed
- *  p	permissions
- *  b	base
- *  l	length
- *  o	offset
- *  t	type
+#if __has_feature(capabilities)
+/*-
```

Comment:
Doesn't this erroneously flag it as a license statement?

Reply:
No, just a comment block that shouldn't be reindented.  License statements are some of the most common comment blocks where that is true, but comments with ASCII art are also comments you don't want to be reindented.  I think it's something the indent(1) tool respects.

Reply:
style(9) says:
> If you desire to flag indent(1) to not reformat a comment that starts in the first column which is not a license or copyright notice, change the dash to a star for those comments.

Reply:
Interesting.  I don't think that rule is consistently followed, at least not on purpose.  Some parts of the tree use doxygen comments which mandate this, but due to doxygen requiring it.